src/picongpu/include/particles/Particles.tpp

```diff
@@ -72,7 +73,9 @@ datasetID( datasetID )
 
     log<picLog::MEMORY > ( "size for all exchange = %1% MiB" ) % ( (double) sizeOfExchanges / 1024. / 1024. );
 
-    const uint32_t commTag = FrameType::CommunicationTag + SPECIES_FIRSTTAG;
+    const uint32_t commTag = PMacc::traits::GetUniqueTypeId<FrameType>::uid + SPECIES_FIRSTTAG;
```

Comment:
uid is:
- a `uint64_t`
- *somewhere* in the `uint64_t` range

therefore this line might overflow already due to two reasons ;) (ignoring the `MPI_TAG_UB` problem for one moment)